packages/react-examples/.storybook/preview-head.html

```diff
@@ -0,0 +1,19 @@
+<!--
```

Comment:
**question:** why do we need do things twice ? manager/preview

Reply:
The CSS escape hatches provided by SB give fine-grained control over the CSS, for both the UI and Docs components that are tagged with class names.
-For Storybook’s UI, it recommends to use `".storybook/manager-head.html"`
-For Storybook Docs, it recommends to use `".storybook/preview-head.html"`

During testing:
-Using only ".storybook/preview-head.html" didn't reflect the favicon or style overrides.
-Using only ".storybook/manager-head.html" does reflect the favicon and style overrides.

Summarizing:
We could do with solely using the ".storybook/manager-head.html" until we want to customize SB Docs to have a different "look and feel".